src/test/java/com/epam/tc/hw5/page/DifferentElementsPage.java

```diff
@@ -56,22 +56,10 @@ public String getDifferentElementsURL() {
         return webDriver.getCurrentUrl();
     }
 
-    @Step("Select Water Checkbox")
-    public void selectWaterCheckbox() {
-        waterCheckbox.click();
-    }
-
-    public void isWaterCheckboxSelected() {
-        assertThat(waterCheckbox.isSelected());
-    }
-
-    @Step("Select Wind Checkbox")
-    public void selectWindCheckbox() {
-        windCheckbox.click();
-    }
-
-    public void isWindCheckboxSelected() {
-        assertThat(windCheckbox.isSelected());
+    @Step("Select checkboxes")
+    public void getElements(String checkboxName) {
```

Comment:
It's not getElmenets! It can be selectElements

Reply:
Changed from from get to selectElements.